### PR TITLE
Best human identifier

### PR DESCRIPTION
**Pull request by dragon-dxw**
Add a "best_human_identifier" to all Document classes for display purposes

### CODE REVIEW

File: src/caselawclient/models/press_summaries.py [hunk 1]
```diff
@@ -27,3 +27,7 @@ def neutral_citation(self) -> str:
                 "akn": "http://docs.oasis-open.org/legaldocml/ns/akn/3.0",
             },
         )
+
+    @property
+    def best_human_identifier(self) -> Optional[str]:
```
Comment (jacksonj04) [inline src/caselawclient/models/press_summaries.py:32]: I don't think this is `Optional`, surely every summary has an NCN?

File: src/caselawclient/models/judgments.py [hunk 2]
```diff
@@ -28,3 +28,7 @@ def neutral_citation(self) -> str:
                 "akn": "http://docs.oasis-open.org/legaldocml/ns/akn/3.0",
             },
         )
+
+    @property
+    def best_human_identifier(self) -> Optional[str]:
```
Comment (jacksonj04) [inline src/caselawclient/models/judgments.py:33]: Is this really `Optional`?

**Review by jacksonj04 — APPROVED** [review]
One query about if NCNs are `Optional`, otherwise 🚢

**Comment by dragon-dxw** [comment]
I'd been fighting the typechecker, and you're absolutely right that I can just do that.

**Pull request merged**